docs/source/advanced_features/fusion_dataset.rst

```diff
@@ -6,10 +6,7 @@ Fusion dataset represents datasets with data collected from multiple sensors.
 Typical examples of fusion dataset are some autonomous driving datasets,
```

Comment:
Please add the following two items in commit message body:
1. Delete unnecessary cross-references.
2. Deprecate "(ref)" style cross-references and hyperlinks.